EIPS/eip-2981.md

```diff
@@ -139,7 +143,11 @@ This EIP mandates the inclusion of the `_data` parameter and the `_royaltyPaymen
 
 ## Backwards Compatibility
 
-This standard is completely compatible with current ERC-721 standards - in fact it requires it.
+This standard is compatible with current ERC-721 and ERC-1155 standards.
+
+## Universal Royalty Payments
+
+Although designed specifically with NFTs in mind, this standard does not require that a contract implementing EIP-2981 is compatible with either ERC-721 or ERC-1155 standards. Any other contract could integrate with this, provided it uniquely identifies assets by ID (like `tokenId`) to return royalty payment information. ERC-2981 is therefore a universal royalty standard for many other asset types.
```

Comment:
>'Any other contract could integrate with this, provided it uniquely identifies assets by ID (like `tokenId`) to return royalty payment information.)'

This is not strictly true and open to misinterpretation.  I suggest a tighter definition:

>'Any other contract could support this interface to signal royalty payment information, with the limitation that assets can only be uniquely identified using the token id argument.'